third_party/terraform/resources/resource_iam_policy.go.erb

```diff
@@ -129,6 +130,9 @@ func setIamPolicyData(d *schema.ResourceData, updater ResourceIamUpdater) error
 	if err != nil {
 		return fmt.Errorf("'policy_data' is not valid for %s: %s", updater.DescribeResource(), err)
 	}
+<% unless version == 'ga' -%>
+	policy.Version = 3
```

Comment:
Do we expect this version to change often? Can we set this as a constant somewhere?

Reply:
Done